app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java

```diff
@@ -30,7 +40,42 @@ public void onCreate() {
```

Comment:
Use PreferenceUtil to read/write from preferneces.
It also exposes getInt/Float that will ease the usage !fix